src/Symfony/Component/HttpKernel/HttpCache/HttpCache.php

```diff
@@ -36,7 +32,7 @@ class HttpCache implements HttpKernelInterface, TerminableInterface
     private $traces = array();
 
     /**
-     * The available options are:
+     * The available options are:.
```

Comment:
Dot must be removed

Reply:
Sf follows PHPDoc here:
https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc.md#51-summary

first statement in docblock here is actually a `description`, while PHPDoc ask us to have `summary` first, that's why it got full stop added by PHP CS Fixer.